packages/tradershub/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useAvailableMT5Accounts } from '@deriv/api';
+import { Provider } from '@deriv/library';
+import { Button, Heading, useBreakpoint } from '@deriv/quill-design';
+import { Modal } from '../../../../components/Modal';
+import { DynamicLeverageContext } from '../../components/DynamicLeverageContext';
+import { Jurisdiction, MarketType, MarketTypeDetails } from '../../constants';
+import { DynamicLeverageScreen, DynamicLeverageTitle } from '../../screens/DynamicLeverage';
+import { JurisdictionScreen } from '../../screens/Jurisdiction';
+
+const JurisdictionModal = () => {
+    const [selectedJurisdiction, setSelectedJurisdiction] = useState('');
+    const [isDynamicLeverageVisible, setIsDynamicLeverageVisible] = useState(false);
+    const [isCheckBoxChecked, setIsCheckBoxChecked] = useState(false);
+
+    const { show } = Provider.useModal();
+    const { getCFDState, setCfdState } = Provider.useCFDContext();
+
+    const { isLoading } = useAvailableMT5Accounts();
+    const { isMobile } = useBreakpoint();
+
+    const marketType = getCFDState('marketType') ?? MarketType.ALL;
+
+    const { title } = MarketTypeDetails[marketType];
+
+    const toggleDynamicLeverage = useCallback(() => {
+        setIsDynamicLeverageVisible(!isDynamicLeverageVisible);
+    }, [isDynamicLeverageVisible, setIsDynamicLeverageVisible]);
+
+    const jurisdictionTitle = `Choose a jurisdiction for your Deriv MT5 ${title} account`;
+
+    const JurisdictionFlow = () => {
+        if (selectedJurisdiction === Jurisdiction.SVG) {
+            return null; // MT5PasswordModal
```

Comment:
I think you can use DummyComponent from features/cfd/components

Reply:
Updated.